pkg/cmd/cli/describe/projectstatus.go

```diff
@@ -376,6 +376,9 @@ func getMarkerScanners(logsCommandName, securityPolicyCommandFormat, setProbeCom
 		func(g osgraph.Graph, f osgraph.Namer) []osgraph.Marker {
 			return deployanalysis.FindDeploymentConfigReadinessWarnings(g, f, setProbeCommandName)
 		},
+		func(g osgraph.Graph, f osgraph.Namer) []osgraph.Marker {
+			return deployanalysis.FindDeploymentConfigLivenessWarnings(g, f, setProbeCommandName)
```

Comment:
> Shouldn't we also be marking pods that are not under a DC? Lone pods, petsets, pods from RCs

We probably don't care about lone pods that aren't part of services, but I don't know how much I care about that use-case.

@juanvallejo Seems like you want to inspect podspec nodes and locate their `GetTopLevelContainerNode`  Might need some de-dupiing logic.  Doesn't look like I did it in `FindUnmountableSecrets`